router/sources/router.move

```diff
@@ -154,17 +154,25 @@ module router::router {
         }
     }
 
+    /// @param user The user who is paying for the registration
+    /// @param domain_name The domain name to register
+    /// @param registration_duration_secs The duration of the registration in seconds
+    /// @param target_addr The address the registered name will point to
+    /// @param to_addr The address to send the token to. If none, then the user will be the owner. Not available in MODE_V1 because token transfers are unreliable (receiver must already have token store)
     public entry fun register_domain(
         user: &signer,
         domain_name: String,
-        registration_duration_secs: u64
+        registration_duration_secs: u64,
+        target_addr: Option<address>,
+        to_addr: Option<address>,
     ) acquires RouterConfig {
         let mode = get_mode();
         if (mode == MODE_V1) {
             assert!(
                 registration_duration_secs % SECONDS_PER_YEAR == 0,
                 error::invalid_argument(ENOT_MULTIPLE_OF_SECONDS_PER_YEAR)
             );
+            assert!(option::is_none(&to_addr), error::not_implemented(ENOT_IMPLEMENTED_IN_MODE));
```

Comment:
why is transfer disabled for v1? i thought it's implemented in `transfer_name` below for v1?